code/game/objects/items/miscellaneous.dm

```diff
@@ -19,3 +19,9 @@
 	icon_state = "skub"
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("skubbed")
+
+/obj/item/skub/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] has declared themself as anti-skub! The skub tears them apart!")
+
+	user.gib()
+	playsound(loc, 'sound/items/eatfood.ogg', 50, 1, -1)
```

Comment:
`loc` => `src`

Reply:
return MANUAL_SUICIDE

Reply:
changed those to the requested stuff, im awful at this tbh